### DISCUSSION TITLE
CS28%40USC/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#105] [Announcements]
# CS28%40USC/

https://csmsapp.org/CS28%40USC/

**Comment by hwj20** [comment]
听说，usc sop里的学校名字写错了都能被录

**Comment by dannydxj** [comment]
正经提问，如果未来想做 research 的话到底要不要接啊？没有别的 offer 了现在。。。在想要不要 gap，但是 gap 一年的话能做的东西现在看来也比较有限。

**Comment by Yangxinyee** [comment]
我看还有个MSAI的项目 这个咋样啊 bar是啥情况呢？

**Comment by chkhu** [comment]
发offer的顺序有讲究吗？听说rank高的先发？

**Comment by chkhu** [comment]
现在改成CS32了

**Comment by JamesSand** [comment]
竟然不用交推荐信，还有这么好的项目？

**Comment by jinxxx-cai** [comment]
usc两年光学费，cs32项目学校按照最低学分要求预估都将近89k美元
